src/platform/tls_schannel.c

```diff
@@ -1172,7 +1252,7 @@ CxPlatTlsSecConfigCreate(
     if (CredConfig->Flags & QUIC_CREDENTIAL_FLAG_ENABLE_OCSP) {
         Credentials->dwFlags |= SCH_CRED_SNI_ENABLE_OCSP;
     }
-    if (CredConfig->Flags & QUIC_CREDENTIAL_FLAG_DEFER_CERTIFICATE_VALIDATION) {
+    if (CredConfig->Flags & QUIC_CREDENTIAL_FLAG_DEFER_CERTIFICATE_VALIDATION && IsClient) {
```

Comment:
Why? Schannel not support it on server?

Reply:
The flag is not supported on server, as far as I can tell (Invalid parameter returned). I can create a new QUIC_CREDENTIAL flag to enable client certificate validation instead of using this one and special-casing it for server.

Reply:
Should we fail the app tries to configure it?